radar/languages-and-frameworks/typescript.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Typescript"
+ring: adopt
+quadrant: "languages-and-frameworks"
+tags: [typescript]
+---
+
+[TypeScript](https://www.typescriptlang.org/) is a superset of JavaScript that adds static typing to the language. It transpiles to JavaScript, meaning that an application written in Typescript will be executed in the a Javascript runtime. The JavaScript runtime, whether it's a browser environment or Node.js, doesn't have direct awareness of the original TypeScript source code. It treats the transpiled JavaScript code just like any other JavaScript code.
```

Comment:
@balanza probably there is a typo "..meaning that an application written in Typescript will be executed in **a** Javascript runtime"

Reply:
What's the typo? I don't see it 🙃

Reply:
sorry! it's a finesse
"..will be executed in **the a** Javascript runtime" should be "..will be executed in **a**Javascript runtime" 
if I intended your sentence correctly

Reply:
Great catch, I really didn't see it thanks

Reply:
Fixed in https://github.com/pagopa/technology-radar/pull/28/commits/68ef2cb3e5dce18bb1a6c3de80ac0c1a0c34c645